### PR TITLE
Add image composite element to feed if user exists

### PR DESCRIPTION
**Pull request by jonathanyeong**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Renders an `<image>` tag in the RSS feed. If the feed is a tag or the latest feed, user won't exist and we will omit this field.

## Related Tickets & Documents
Closes https://github.com/forem/forem/issues/9934

## QA Instructions, Screenshots, Recordings
User feed

Tag feed

All Feed

**To test**:
- Login as a user with no organization.
- Navigate to `localhost:3000/feed`
  - No <image> tag should exist.
- Navigate to `localhost:3000/rss`
  - No <image> tag should exist.
- Navigate to `localhost:3000/feed/tag/<tagname>`
  - No <image> tag should exist.
- Navigate to `localhost:3000/feed/<username>`.
  - <image> tag should exist. Made up of <url>, <title>, <link> elems.

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Comment by jonathanyeong** [comment]
Totally taking a stab in the dark. Draft PR for now - I'm still trying to get everything setup locally 😃.

**Comment by jonathanyeong** [comment]
@rhymes what are thoughts on the rubocop block length error? Should we disable block length for the `feed.rss.builder`? I could use partials, but I read on SO that there's issues with identing: https://stackoverflow.com/questions/2986245/ruby-on-rails-using-xml-builder-partials.

File: app/views/articles/feed.rss.builder [hunk 1]
```diff
@@ -8,7 +8,13 @@ xml.rss version: "2.0" do
     xml.description user ? user.summary : SiteConfig.community_description
     xml.link user ? app_url(user.path) : app_url
     xml.language "en"
-
+    if user
+      xml.image do
+        xml.url user.profile_image_90
+        xml.title user.name
```
Comment (rhymes) [inline app/views/articles/feed.rss.builder:14]: This should probably be `#{user.username} profile image`

**Review by rhymes — CHANGES_REQUESTED** [review]
Hi @jonathanyeong, thanks for tackling this, sorry for the late reply! 

I think we should add a few tests, based on my notes here https://github.com/forem/forem/issues/9934#issuecomment-703224961

**Comment by jonathanyeong** [comment]
Gotcha makes sense! Having a look at moving it into a partial. Hopefully, dealing with the indentation isn't terrible!

**Review by rhymes — APPROVED** [review]
Hi @jonathanyeong! 

I manually tested the code and ran the tests locally and all works great. The build is temporarily having issue that will be resolved soon.

In the meantime I went ahead and approved :D

**Comment by jonathanyeong** [comment]
@rhymes woohoo thank you so much! This is my first forem pr!

**Comment by rhymes** [comment]
@jonathanyeong welcome to the project :))

**Review by juliannatetreault — APPROVED** [review]
Thanks so much for taking the time to contribute to Forem, @jonathanyeong! I tested your PR locally and everything works as expected! I also really appreciate the tests that you added around your changes--this is great. 🎉

**Pull request merged**